src/utils/__tests__/getFlowType-test.js

```diff
@@ -232,6 +232,39 @@ describe('getFlowType', () => {
     }, raw: '{ a: string, b: ?xyz }'});
   });
 
+  it('handles ObjectTypeSpreadProperty', () => {
+    var typePath = statement(`
+      var x: { ...OtherProps, ...MyType, a: string };
+
+      type MyType = { myString: string };
+    `).get('declarations', 0).get('id').get('typeAnnotation').get('typeAnnotation');
+
+    expect(getFlowType(typePath)).toEqual({
+      name: 'signature',
+      type: 'object',
+      signature: {
+        inheritedProperties: [
+          { name: 'OtherProps', value: { name: 'OtherProps' } },
```

Comment:
```
import type { BarType as Foo } from 'bar';
```

```
{ ...Foo }
```

```
{ 
module: 'bar',
name: 'BarType'
}
```


---------------

```
import type URI from 'URI';
```

```
{ ...URI }
```

```
{ 
module: 'URI',
name: 'default'
}
```

Reply:
I like that. Not sure if the local identifier is worth having in here too, 

```js
{ 
module: 'bar',
name: 'BarType',
local: 'Foo'
}
```

Reply:
@danez: I originally thought that anything that's local wouldn't be of interest to someone reading the documentation. But who knows, maybe it's useful for the person that generates the documentation.